io.gpl.wxmaxima.yml

```diff
@@ -0,0 +1,71 @@
+app-id: io.gpl.wxmaxima
```

Comment:
`gpl.io` seems to just be a parked domain. Is it under the control of the upstream developer of that project?

Reply:
According to this 
https://github.com/wxMaxima-developers/wxmaxima/blob/Version-23.10.0/data/io.github.wxmaxima_developers.wxMaxima.desktop

this is just wrong.

Reply:
I do not have control of the wxmaxima domain so I chose an arbitrary id. Should I change the id to io.github.wxmaxima_developers.wxMaxima? Do I have to do a new pull request?

Reply:
> so I chose an arbitrary id.

Definitely not.

Use upstream. It's all setup for that it seems.

> Do I have to do a new pull request?

Why?